itest/lnd_channel_force_close_test.go

```diff
@@ -528,19 +548,17 @@ func channelForceClosureTest(ht *lntest.HarnessTest,
 	// one.
 	ht.AssertNumPendingSweeps(alice, numInvoices)
 
-	// Mine a block to trigger the sweeps.
-	ht.MineEmptyBlocks(1)
-
-	// Wait for them all to show up in the mempool and expect the timeout
-	// txs to be aggregated into one.
-	htlcTxIDs := ht.Miner.AssertNumTxsInMempool(1)
+	// Wait for them all to show up in the mempool
+	//
+	// NOTE: after restart, all the htlc timeout txns will be offered to
+	// the sweeper with `Immediate` set to true, so they won't be
```

Comment:
due to the locktime split, they won't be aggregated besides the immediate flag ?

Reply:
nope, they share the same locktimes, so it's only the immediate flag that's keeping them apart.